numpy/_core/src/multiarray/compiled_base.c

```diff
@@ -1602,7 +1602,6 @@ pack_inner(const char *inptr,
         npy_intp vn_out = n_out - (remain ? 1 : 0);
         const int vstep = npyv_nlanes_u64;
         const int vstepx4 = vstep * 4;
-        const int isAligned = npy_is_aligned(outptr, sizeof(npy_uint64));
```

Comment:
Why are you removing this?

Reply:
compiler warns on `-Wunused-but-set-variable` in Clang